Classification/include/CGAL/Classification/Mesh_feature_generator.h

```diff
@@ -186,32 +188,13 @@ class Mesh_feature_generator
       t.start();
 
       if (lower_grid == nullptr)
-        grid = new Planimetric_grid (range, point_map, bbox, this->voxel_size);
+        grid = std::make_shared<Planimetric_grid> (range, point_map, bbox, this->voxel_size);
       else
-        grid = new Planimetric_grid(lower_grid);
+        grid = lower_grid->get_ptr();
```

Comment:
This lines changes the semantic of the code.

`Planimetric_grid (Planimetric_grid* lower_scale)` creates a new grid, with twice the resolution of the grid pointed by `lower_scale`.

@sgiraudot Do you confirm?

Reply:
Indeed. Anyway I oppose the changes in classification (see my comment below).